homeassistant/components/gpstracker/const.py

```diff
@@ -0,0 +1,52 @@
+"""Constants for the GPS Tracker integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+
+from gps_tracker.client.datatypes import TrackerIcon
+
+CLIENT = "client"
+
+DOMAIN = "gpstracker"
+
```

Comment:
I am not sure that gpstracker is the best domain to use. Maybe invoxia or invoxia_gps is better.

Reply:
I initially planned to name the integration `invoxia` but this is a registred trademark (hence the ™ I use everywhere) and I do not wan't to let anyone think this integration is official. This is why I came up whis another domain but yeah, I have also troubles with `gpstracker`, it gives the impression that it will support GPS Trackers from many different providers...

Do you happen to know if there has been issue in the past with integrations named after a trademark without the manufacturer approval?

Reply:
Please adjust the domain to `invoxia`.

Reply:
Ok this is done in d6da24e68fa6a8d0a80d96e9d9b2aea61a010d63. I still need to update documentation and branding.